testing/integration_testing/restore_secrets.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script for restoring secrets into the integration test projects.
+
+Usage:
+
+python restore_secrets.py --passphrase <passphrase> --repo_dir <path_to_repo>
+
+repo_dir refers to the C++ SDK github repository. Defaults to current directory.
+
+As an alternative to passing the passphrase as a flag, the password can be
+stored in a file and passed via the --passphrase_file flag.
```

Comment:
nit: Let's list these in a more structured way:
python restore_secrets.py --passphrase <passphrase> [--repo_dir <path>]
python restore_secrets.py --passphrase_file <path> [--repo_dir <path>]

--passphrase <passphrase>: Passphrase to decrypt the files. This option is insecure on a multi-user machine; use the --passphrase_file option instead.
--passphrase_file <path>: Specify a file to read the passphrase from (only reads the first line).
--repo_dir <path>: Path to the C++ SDK github repository. Defaults to current directory.

Reply:
(also you should make sure it only reads the first line of passphrase_file